frontend/src/components/podcast/components/episode-table/components/link-episode.tsx

```diff
@@ -0,0 +1,17 @@
+import { AppRoute } from 'common/enums/enums';
```

Comment:
not a bad idea, but too much to create a separate component for this. better describe it where you use

Reply:
also, each component should be in the separate folder